src/ring/server/standalone.clj

```diff
@@ -99,7 +99,7 @@
          (addShutdownHook (Thread. destroy))))
     (try-port (port options)
       (fn [port]
-        (let [options (merge {:port port} options)
+        (let [options (assoc options :port port)
```

Comment:
It looks like you're making the port passed to the lambda take precedence over what's in the options, which looks sensible to me -- we're just using the precedence rules in https://github.com/PawelStroinski/ring-server/blob/b3e1bc2354decd7181f85c079d27606d9709cdf5/src/ring/server/options.clj#L9 so options still gets to have an effect.